tensorflow/docs_src/programmers_guide/saved_model.md

```diff
@@ -92,6 +92,11 @@ with tf.Session() as sess:
   print("v2 : %s" % v2.eval())
 ```
 
+Notes:
+
+*  There is not a physical file called "/tmp/model.ckpt". It is the **prefix** of filenames created
```

Comment:
@tedhtchang this line length seems a bit high compared to previous lines in the file so can you make these three lines.

Reply:
Ok. change made :)